data/exclusions-disease-gene.tsv

```diff
@@ -0,0 +1,8 @@
+omim_id	mondo_id	mondo_label	orcid	exclusion_reason_comment
+OMIM:603956	MONDO:0002974	cervical cancer'	https://orcid.org/0000-0002-4142-7153	evidence of various genes involved
+OMIM:619151	MONDO:0030894	"AMED syndrome, digenic'"	https://orcid.org/0000-0002-4142-7153	digenic
```

Comment:
To explain the HGNC removals in the diff.diff file using this as an example. When creating the omim.owl file from the omim.ttl file, there is a step in the make goal that adds the hgnc links where there is a disease2gene association represented with RO:0004003. Since this disease no longer has the RO:0004003 association and the gene ADH5 is not used as a disease-defining gene for any other disease it does not get converted to a HGNC identifier. The same thing happens with `OMIM:620040	MONDO:0031057	"dyskeratosis congenita, digenic'"`.

If you open the omim-b4.owl and omim-after.owl files in Protege and go to the Object properties tab and check the Usage of RO:0003302 it's easy to see where the new associations exist that use the property (the number is small enough to not need to sparql this information). And looking at the gene entry in Protege, you can also check which of these genes that now have the RO:0003302 association also still have other disease-defining associations for other diseases.

Reply:
Really glad you looked into the HGNC stuff! Thanks. I was thinking that might be the case.

Also, good Protégé tips!